sky/templates/lambda-ray.yml.j2

```diff
@@ -24,13 +24,17 @@ available_node_types:
   ray_head_default:
     resources: {}
     node_config:
+      AuthorizedKey: |
```

Comment:
Is this interpreted/used anywhere?

Reply:
Good catch! That is a remnant. Removed. Thanks!